ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/port-channel-interfaces.md

```diff
@@ -180,6 +184,29 @@ interface Ethernet50
 | Port-Channel106 | bpdu enabled | switched | access | - | - | - | - | - | - | - |
 | Port-Channel107 | bpdu true | switched | access | - | - | - | - | - | - | - |
 | Port-Channel108 | bpdu false | switched | access | - | - | - | - | - | - | - |
+| Port-Channel109.1 | TENANT_A pseudowire 1 interface | l2dot1q | access | - | - | - | - | - | - | - |
+| Port-Channel109.100 | TENANT_A pseudowire 2 interface | l2dot1q | access | - | - | - | - | - | - | - |
+| Port-Channel109.200 | TENANT_A pseudowire 3 interface | l2dot1q | access | - | - | - | - | - | - | - |
+| Port-Channel109.300 | TENANT_A pseudowire 4 interface | l2dot1q | access | - | - | - | - | - | - | - |
+| Port-Channel109.400 | TENANT_A pseudowire 3 interface | l2dot1q | access | - | - | - | - | - | - | - |
+| Port-Channel109.1000 | L2 Subinterface | l2dot1q | access | - | - | - | - | - | - | 0000:0000:0303:0202:0101 |
```

Comment:
Should mode be "access" on these? We could set mode to "flexencap" possibly. Otherwise LGTM!

Reply:
We should not even show them here under L2, but I forgot to exclude them in the port-channel template. Fixed now.

Reply:
Gotcha. Ok LGTM!